gnomad/utils/annotations.py

```diff
@@ -2244,8 +2243,16 @@ def _create_group_dicts(
         :return: Dictionary containing variant frequency information,
             - (by genetic ancestry group and sex if desired) for specified variant.
         """
+        if group_sex:
+            cohort_id = f"{group_id.upper()}.{group_sex}"
+            freq_index_key = f"{group_id}_{group_sex}_adj"
+        else:
+            cohort_id = f"{group_id.upper()}"
+            freq_index_key = f"{group_id}_adj"
+        record_id = f"{gnomad_id}.{cohort_id}"
```

Comment:
looks like a bit of code - does this speed up performance?

Reply:
I think the reason I did this is so all the `freq` keys are created in this inner function instead of where the function is called. They are all based on the group_id without sex information. Previously it was sending both a group_id including sex, and the sex label itself. And now that we are also pulling out the `hemizogytes`, we need the group_id without sex in order to make the key with XY.

Reply:
👍